perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -511,6 +513,30 @@ sub admin_url {
     return $cobrand->admin_base_url . '/report_edit/' . $self->id;
 }
 
+=head2 tokenised_url
+
+Return a url for this problem report that logs a user in
+
+=cut
+
+sub tokenised_url {
```

Comment:
I get what you're saying, but the issue there is that at the moment we've got dynamic attributes (namely `$user` and `$params`) that get passed in too. Is there a way round that?